.github/workflows/main.yaml

```diff
@@ -257,9 +267,17 @@ jobs:
         uses: ./.github/actions/gn
       - name: Build Cobalt
         uses: ./.github/actions/build
+      - name: 'Check Artifact Size'
+        uses: ./.github/actions/check_artifact_size
+        if: ${{ (inputs.check_artifact_size && needs.initialize.outputs.evergreen_loader == 'null') || (inputs.check_artifact_size && needs.initialize.outputs.evergreen_loader != 'null' && matrix.sb_api_version == '') }}
```

Comment:
This can be simplified slightly to `${{ inputs.check_artifact_size && (needs.initialize.outputs.evergreen_loader == 'null' || matrix.sb_api_version == '')`

Side question, should this action run if `evergreen_loader` is `null`?

Reply:
Updated the action to be ran for all Evergreen versions.